nextflow_schema.json

```
@@ -488,6 +488,11 @@
                     "description": "Database for taxonomic binning with kraken2.",
                     "help_text": "The database file must be a compressed tar archive that contains at least the three files `hash.k2d`, `opts.k2d` and `taxo.k2d`. E.g. ftp://ftp.ccb.jhu.edu/pub/data/kraken2_dbs/minikraken_8GB_202003.tgz."
                 },
+                "krona_db": {
+                    "type": "string",
+                    "description": "Database for taxonomic binning with krona",
+                    "help_text": "Local `taxonomy.tab` file for Krona, instead of downloading. Point at the `.tab` file."
```

This doesnt need to be local, a path to a remote source can work as well.
```suggestion
                    "help_text": "Path to `taxonomy.tab` file for Krona, instead of downloading the default file. Point at the `.tab` file."
```